registries/_format/decimal.md

```diff
@@ -0,0 +1,31 @@
+---
+owner: baywet
+issue: 889
+description: A fixed point decimal number
```

Comment:
I'd say "A fixed point decimal number of unspecified precision and range"

That is a concise way to communicate the difficulty of this format.

Reply:
added. thanks